src/style.css

```
@@ -0,0 +1,162 @@
+* {
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
+.page {
+  display: flex;
+  flex-direction: column;
+  max-width: 198px;
+  max-height: 406px;
+  --left-gap: 16px;
+  --right-gap: 16px;
+}
+
+.card {
+  justify-content: center;
+  align-items: center;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 0;
+}
+
+.card__texts {
+  display: flex;
+  align-content: stretch;
+  margin: 40px var(--right-gap) 0 var(--left-gap);
+}
+
+.card__texts-text {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__small-text {
+  display: flex;
+  margin: 4px var(--right-gap);
+}
+
+.card__small-textcode {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.star {
+  display: flex;
+  flex-direction: row;
+  margin: 16px var(--right-gap) 0 var(--left-gap);
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars-star {
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  border-radius: 0.5px;
```

```suggestion
  border-radius: 1px;
```
do not use non-integer px values - it is not supported by all browsers